Lesson2/les2.6.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+        "use strict"
+        
+        /*
+        6**. (Сложное задание, требует времени и возможно гугления, делайте по желанию.) Программа должна спросить у пользователя число, это будет количество денег, которое он хочет положить на счет в банке. Затем программа должна выдать примерно такое сообщение: 
+        "Ваша сумма в 101 рубль успешно зачислена." - в случае если пользователь ввел 101. 
+        "Ваша сумма в 10020 рублей успешно зачислена." - в случае если пользователь ввел 10020. 
+        "Ваша сумма в 120104 рубля успешно зачислена." - в случае если пользователь ввел 120104. 
+        То есть ваша задача выводить слово «рубль» в правильном падеже, в зависимости от введенного числа.
+        */
+        
+        let ruble = Number(prompt("Какое количество денег вы хотите положить на счет в банк"));
+        let last_numb = ruble % 10
```

Comment:
```suggestion
        let last_numb = ruble % 10
```
Беру последнюю цифру остатком от деления.